CHANGELOG.txt

```diff
@@ -1,6 +1,20 @@
 BlocklyDuino
 =======================
 
+1.1 2015/02/07
```

Comment:
There are no new blocks or deleted blocks.  All previously saved programs should still load and should generate identical code as before.  I don't think there are any incompatibilities created here.